snake.asm

```diff
@@ -46,12 +46,9 @@ je .food                ; If it's food, place new food and skip tail removal
 es lodsw                ; Load the previous position (tail) from the stack into ax
 ; The instruction above retrieves the last position of the snake's tail from the stack,
 ; using the fact that si was set to point to the stack (mov si, sp)
-; After loading the value with lodsw, the stack pointer (sp) needs manual adjustment
-; to "pop" the read value, ensuring proper stack management
 xchg ax, bx             ; Swap the values of ax and bx for tail removal
 mov [bx], ah            ; Erase the tail character at the memory location indicated by bx
-; This line updates the display by replacing the tail character at its previous position
-; with an invisible character. This process is vital for simulating the snake's movement;
-; as the snake advances, its tail is effectively removed from its previous location,
+; Replacing the tail character at its previous position with an invisible character
+; as the snake advances, its tail is removed from its previous location,
 ; creating the illusion of continuous motion, (just like in original snake game)

```

Comment:
Again, you can simply remove the comments in lines 51-53 and instead explain that the bh which is invisible in cp437 is being loaded into ah